src/target/ext/host_io.rs

```diff
@@ -9,6 +9,7 @@ bitflags! {
     ///
     /// Extracted from the GDB documentation at
     /// [Open Flags](https://sourceware.org/gdb/current/onlinedocs/gdb/Open-Flags.html#Open-Flags)
+    /// [LLDB-specific Flags](https://github.com/llvm/llvm-project/blob/ec642ceebc1aacc8b16249df7734b8cf90ae2963/lldb/include/lldb/Host/File.h#L47-L66)
```

Comment:
Sorry to keep throwing nits your way, but if you run `cargo doc`, you'll find that the documentation will look quite weird the way its currently written:

> Extracted from the GDB documentation at *Open Flags* *LLDB-specific Flags*

I'd probably do something like:

```rust
    /// Extracted from the GDB documentation at
    /// [Open Flags](https://sourceware.org/gdb/current/onlinedocs/gdb/Open-Flags.html#Open-Flags),
    /// and the LLDB source code at 
    /// [`lldb/include/lldb/Host/File.h`](https://github.com/llvm/llvm-project/blob/ec642ceebc1aacc8b16249df7734b8cf90ae2963/lldb/include/lldb/Host/File.h#L47-L66)
```

Reply:
Np, I should have checked it out. Fixing

Reply:
Done. It looks much better now :)